src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -81,30 +106,22 @@ case class Run(name: String, pipeline: Pipeline, tag: String) {
       // just use the state names.
       val prevStateName = previousStatus map { _.toString } getOrElse "-"
       Log.info(s"$tag: Status change from $prevStateName to $currentStatus")
-    }
 
-    currentStatus match {
-      case x: TerminalRunStatus => x
-      case _ =>
-        Thread.sleep(5000)
-        waitUntilComplete(Option(currentStatus))
+      // Update the database state:
+      val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(jesId)), Option(JesStatus(currentStatus.toString)))
+      dataAccess.updateExecutionBackendInfo(workflowId, call, newBackendInfo)
     }
-  }
 
-  @tailrec
-  final def waitUntilRunningOrComplete(): Unit = {
-    val currentStatus = status()
-    currentStatus match {
-      case Initializing => // Done
-      case x: TerminalRunStatus => // Done
-      case _ =>
-        Thread.sleep(5000)
-        waitUntilRunningOrComplete()
+    if(breakout(currentStatus)) {
+      currentStatus
+    } else {
+      Thread.sleep(1000)
```

Comment:
Oh, initialization seemed to be taking a long time so I tried reducing polling interval to see if anything was improved.

Ideally we'd want some kind of polling-time back-off so short jobs are ended quickly but long-running (hours... or days?) jobs don't get polled every few seconds.